cachi2/core/package_managers/yarn/main.py

```diff
@@ -47,7 +47,8 @@ def _resolve_yarn_project(project: Project, output_dir: RootedPath) -> list[Comp
         if project.is_zero_installs:
             _check_yarn_cache(project.source_dir)
         else:
-            _fetch_dependencies(project.source_dir, output_dir)
+            # fetch the dependencies
```

Comment:
I like that we simplified (and fixed) the code with these changes, but I don't like that we lost the context that the function `_fetch_dependencies` brought, and now we rely on a comment to convey it.

I don't have a good suggestion to fix it though, so this is just me nitpicking :shrug:

Reply:
I can put it back, I don't have a preference really; since it's quite common practice to have one-liner instance methods just to convey semantic meaning, despite me not liking the same for regular functions, sure, why not, I don't really have a good argument against such practice (I'm afraid unnecessary calls and stack operations wouldn't hold and would also be inconsistent with the practice for methods anyway) :)